src/components/ResultsItem/ResultsItem.tsx

```diff
@@ -60,6 +61,13 @@ export const ResultsItem = ({
               &nbsp;
             </span>
           )} */}
+          {/* hasType is a flag to decide if we want to show a type */}
```

Comment:
Do we actually need this prop? We can either pass in the type or not pass it and then simply have

``` 
{meta.type && (
  ...
)}

Reply:
I only added it because of the comment about types for search results page being not correctly displayed. I didn't want to introduce any bugs to that page. I can remove it if you think we don't need this. :)

Reply:
Keep it in for now - it shouldn't affect existing results right?

Moving forward we need to get consensus from the product managers on how these items will be labelled and we can always remove if this is handled differently